plugins/woocommerce-admin/client/products/product-variation-form.tsx

```diff
@@ -27,13 +30,15 @@ export const ProductVariationForm: React.FC< {
 					<>General</>
 				</ProductFormTab>
 				<ProductFormTab name="pricing" title="Pricing">
-					<>Pricing</>
+					<PricingSection />
 				</ProductFormTab>
 				<ProductFormTab name="inventory" title="Inventory">
-					<>Inventory</>
+					<ProductInventorySection />
 				</ProductFormTab>
 				<ProductFormTab name="shipping" title="Shipping">
-					<>Shipping</>
+					<ProductShippingSection
+						product={ productVariation as PartialProduct }
```

Comment:
I think this is okay for the time being, but we'll probably need to pass both the variation and product parent to inherit parent properties.

That can wait until https://github.com/woocommerce/woocommerce/issues/35989 to implement this.